tensorboard/webapp/core/types.ts

```diff
@@ -30,3 +30,7 @@ export enum PluginsListFailureCode {
 export const TB_BRAND_NAME = new InjectionToken<string>(
   'TensorBoard brand name'
 );
+
+export interface URLDeserializedState {
+  pluginQueryParams: Record<string, string>;
```

Comment:
I actually considered `unknownQueryParams` but chose this due to the clear usecase. However, since you had the same thought I will switch to `unknownQueryParams`